libs/groups/src/groups.controller.ts

```diff
@@ -0,0 +1,43 @@
+import { Controller, Post, Body } from '@nestjs/common';
+import { IsString, IsOptional, IsArray } from 'class-validator';
+import { Auth } from '@utils/decorators';
+import { ObjectID } from 'mongodb';
+import {
+  ApiTags,
+  ApiBody,
+  ApiProperty,
+  ApiPropertyOptional,
+} from '@nestjs/swagger';
+import { GroupsService } from './groups.service';
+import { Group } from './groups.model';
+
+class CreateGroupDto {
+  @IsArray()
+  @IsString({ each: true })
+  @ApiProperty()
+  public managersUserIds: ObjectID[];
```

Comment:
You can't expect to have ObjectID in DTOs

Reply:
use @ApiProperty({type: String}) this is the only case you need ApiProperty if it didn't work use @Type()